src/pages/item/item-actions/ItemActions.tsx

```diff
@@ -55,6 +134,23 @@ export default function ItemActions(props: { itemId: string | number }) {
 				)}
 				{data && ` (${data.downstreamReferences.length})`}
 			</button>
+			<button
+				className={`button button-tertiary ${
+					isLoading ? 'button-loading button-loading-primary' : ''
+				}`}
+				onClick={showDependenciesHandler}
+				disabled={showDependenciesDisabled}
+				data-test="show-dependency"
+				title="Show Dependency & Associations"
+			>
+				{!isLoading && (
+					<>
+						<span className="icon-add-row-bottom"></span>
+						<span>Show Dependency & Associations</span>
+					</>
+				)}
+				{data && ` (${dependenciesOnBoardCount})`}
```

Comment:
This is similar to the `loading` issue - by the time you have `data`, you don't yet know how large the `dependenciesOnBoardCount` is, you only just start calculating that.  

You got some options here I think:  
1. Display the element only if `dependenciesOnBoardCount` is truthy
2. Display the element whenever the text in the button is displayed too, because that's when everything should be loaded anyway, right?

Reply:
used the new relationsLoading state again